app/main.py

```diff
@@ -1,3 +1,65 @@
-def shop_trip():
-    # write your code here
-    pass
+from datetime import datetime
+import json
+
+from app.shop import Shop
+from app.customer import Customer
+
+
+def shop_trip() -> None:
+    with open("config.json") as config:
```

Comment:
Ensure that the path to `config.json` is correct. This is crucial for the application to load the configuration data properly.